alf/algorithms/td_loss.py

```diff
@@ -146,6 +146,18 @@ def compute_td_target(self, info: namedtuple, target_value: torch.Tensor):
                 td_lambda=self._lambda)
             returns = advantages + target_value[:-1]
 
+        disc_ret = ()
+        if hasattr(info, "discounted_return"):
+            disc_ret = info.discounted_return
+        if disc_ret != ():
+            with alf.summary.scope(self._name):
+                episode_ended = disc_ret > self._default_return
```

Comment:
Where is this `self._default_return` set? By default it's not in `__init__`. If it's only set in a special case, then I think this discounted return thing shouldn't appear here in a general TDLoss. Maybe should drive a subclass from it.